### PR TITLE
simulator/cases: import-data only count the newly regions of the table

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve?
we count the old regions in the statistics for the table, This is not the result I want.

### Check List

Tests

- Manual test (add detailed scripts or steps below)
```
❯ ./bin/pd-simulator -regionNum=10000 -case="import-data" -simLog=info
["import data information"]:
[table-leader="1001 leader: [store 1]:97.60% [store 4]:2.40%"] 
[table-peer="3003 peer:  [store 1]:32.53% [store 2]:0.60% [store 3]:0.50% [store 4]:32.43% [store 5]:0.20% [store 6]:0.23% [store 7]:0.30% [store 8]:0.43% [store 9]:32.43% [store 10]:0.33%"]
[total-leader="11000 leader: [store 1]:9.13% [store 2]:9.13% [store 3]:9.06% [store 4]:9.15% [store 5]:9.14% [store 6]:9.11% [store 7]:9.05% [store 8]:9.12% [store 9]:9.14% [store 10]:9.11%"]
[total-peer="33000 peer: [store 1]:10.08% [store 2]:9.97% [store 3]:9.97% [store 4]:10.08% [store 5]:9.97% [store 6]:9.97% [store 7]:9.97% [store 8]:9.97% [store 9]:10.08% [store 10]:9.97%"]

OK [import-data] total iteration: 1001, time cost: 1m46.775680649s
Add Peer (task) 0
Remove Peer (task) 2202
Add Learner (task) 2212
Promote Learner (task) 2207
Transfer Leader (task) 73
Merge Region (task) 0
Send Maximum (snapshot) 257
Receive Maximum (snapshot) 412
Send Minimum (snapshot) 156
Receive Minimum (snapshot) 261
```

### CODE REVIEW

File: pkg/codec/codec.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 var (
 	tablePrefix  = []byte{'t'}
 	metaPrefix   = []byte{'m'}
-	recordPrefix = []byte{'r'}
+	recordPrefix = []byte("_r")
```
Comment (ZenoTan) [inline pkg/codec/codec.go:26]: Is it necessary to change that?

**Comment by ZenoTan** [comment]
Rest LGTM.

File: pkg/codec/codec.go (hunk 1 above)
Comment (nolouch) [reply pkg/codec/codec.go:26]: yes, it's same as TiDB. https://github.com/pingcap/tidb/blob/master/tablecodec/tablecodec.go#L45-L47

File: tools/pd-simulator/simulator/cases/import_data.go [hunk 2]
```diff
@@ -61,14 +61,14 @@ func newImportData() *Case {
 	simCase.TableNumber = 10
 	// Events description
 	e := &WriteFlowOnSpotDescriptor{}
-	table2 := string(codec.EncodeBytes(codec.GenerateTableKey(2)))
-	table3 := string(codec.EncodeBytes(codec.GenerateTableKey(3)))
+	table12 := string(codec.EncodeBytes(codec.GenerateTableKey(12)))
+	table13 := string(codec.EncodeBytes(codec.GenerateTableKey(13)))
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/cases/import_data.go:65]: why change it

File: tools/pd-simulator/simulator/simutil/key.go [hunk 3]
```diff
@@ -122,6 +122,7 @@ func GenerateTiDBEncodedSplitKey(start, end []byte) ([]byte, error) {
 	}
 
 	var err error
+
```
Comment (lhy1024) [inline tools/pd-simulator/simulator/simutil/key.go:125]:
```suggestion
```

File: tools/pd-simulator/simulator/cases/import_data.go (hunk 2 above)
Comment (nolouch) [reply tools/pd-simulator/simulator/cases/import_data.go:65]: default the data will generator to table0...9.see the details: https://github.com/tikv/pd/blob/master/tools/pd-simulator/simulator/simutil/key.go#L60-L73(default table count is 10)

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**